php/class-customize-snapshot-manager-back-compat.php

```diff
@@ -106,7 +101,7 @@ public function enqueue_controls_scripts() {
 			'initialServerDate' => current_time( 'mysql', false ),
 			'initialServerTimestamp' => floor( microtime( true ) * 1000 ),
 			'theme' => $this->original_stylesheet,
-			'previewingTheme' => isset( $preview_url_query_vars['theme'] ) ? $preview_url_query_vars['theme']: '',
+			'previewingTheme' => ! $this->customize_manager->is_theme_active(),
```

Comment:
Thanks for pointing this out. So this can be fixed by reverting back to:

```php
'previewingTheme' => isset( $preview_url_query_vars['theme'] ) ? $preview_url_query_vars['theme'] : '',
```
?